Dockerfile.ubuntu

```diff
@@ -1,12 +1,13 @@
 FROM ubuntu:bionic
 LABEL maintainer="sig-platform@spinnaker.io"
 
-ENV KUSTOMIZE_VERSION=3.8.6
```

Comment:
See other comment - we shouldn't remove this.  Maybe look at bumping the 3. version of kustomize if anything, but caution flag as we've seen that cause breakage...